llama_hub/library.json

```diff
@@ -1,4 +1,8 @@
 {
+  "AgentSearch": {
```

Comment:
I think you need to rename `AgentSearch` here to match the class name -- `AgentSearchReader`

Reply:
got it, ty